apps/web/src/components/ui/history-table.tsx

```diff
@@ -0,0 +1,130 @@
+import { HeaderLeft } from "@/components/header-left.tsx";
+import { Button } from "@/components/ui/button.tsx";
+import {
+	Table,
+	TableBody,
+	TableCaption,
+	TableCell,
+	TableFooter,
+	TableHead,
+	TableHeader,
+	TableRow,
+} from "@/components/ui/table";
+import useExportToCsv from "@/hooks/useExportToCSV.ts";
+import { HocusPocusProvider } from "@/hooks/useHocuspocus.tsx";
+import { RealtimeProvider, VotingHistory } from "@/hooks/useRealtime.tsx";
+import { state } from "@/store.ts";
+import { ydoc } from "@/yjsDoc.ts";
+import { useParams } from "react-router-dom";
+import { useSnapshot } from "valtio";
+
+// prepare data to be exported, join the votes into a string
```

Comment:
```suggestion
/* prepare data to be exported, join the votes into a string */
```

Can we use tsdoc for functions description?